README.md

```diff
@@ -31,13 +31,13 @@ zone.  This role has a trust relationship with the users account.
 | aws.route53resourcechange | ~> 4.9 |
 | terraform | n/a |
 
-## Modules ##
+## Modules
 
 | Name | Source | Version |
 |------|--------|---------|
 | read\_terraform\_state | github.com/cisagov/terraform-state-read-role-tf-module | n/a |
 
-## Resources ##
+## Resources
```

Comment:
Revert this change that terraform-docs makes:
```suggestion
## Resources ##
```

Reply:
Fixed in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/103/commits/5de085cd1da6a42095891d88144f4bf0b14543a4.